mantra/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "mantrachain",
+  "assets": [
+    {
+      "description": "The native token of MANTRA Chain",
+      "denom_units": [
+        {
+          "denom": "uaum",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "aum",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uaum",
+      "name": "MANTRA Chain",
+      "display": "AUM",
+      "symbol": "AUM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mantra/images/mantra.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mantra/images/mantra.svg"
+      },
+      "coingecko_id": "osmosis",
```

Comment:
CoinGecko id is incorrect here

Reply:
As we do not yet have a Coingecko ID, I have left this blank.